source/includes/faq.md

```diff
@@ -0,0 +1,39 @@
+# Frequently Asked Questions (FAQ)
```

Comment:
Seems this is unneeded as well. At least most of the questions are already answered above, and some seem to be gimmicky.